app/controllers/schedule_chains_controller.rb

```diff
@@ -112,6 +112,21 @@ def new
 
   def create
     @schedule = ScheduleChain.new(params[:schedule_chain])
+    if @schedule.donors.first.is_farmer_market
+      donor_market =   @schedule.donors.first
+      food_type_id = FoodType.where('name'=>"Food")[0].id
+      @schedule.schedules.each do |sch|
+        if sch.location.location_type != 0 && sch.location.is_farmer_market
+          sch.schedule_parts.destroy_all
+          donor_market.location_admins.each do |vendor|
+            sch.schedule_parts.build(food_type_id: food_type_id, location_admin_id: vendor.id).save
+          end
+          sch.schedule_parts.where('location_admin_id':nil).destroy_all
```

Comment:
this syntax looks a little funny to me: `.where('location_admin_id':nil)`, I think we'll at least want a space between before the nil, but I can't tell where the `:` would go, or if the single quotes are needed around `location_admin_id`.